packages/tests/src/e2e/frontend/TestCreateTab.tests.ts

```diff
@@ -50,7 +51,9 @@ describe("Create single tab", function () {
         await CliHelper.createProjectWithCapability(
           appName,
           testFolder,
-          Capability.M365SsoLaunchPage
+          Capability.M365SsoLaunchPage,
+          process.env,
```

Comment:
use `{...process.env, TEAMSFX_CLI_DOTNET: "false"}` instead of using `process.env` directly.